.github/CODEOWNERS

```diff
@@ -85,9 +85,6 @@
 /x-pack/test/functional/apps/machine_learning/  @elastic/ml-ui
 /x-pack/test/functional/services/machine_learning/  @elastic/ml-ui
 /x-pack/test/functional/services/ml.ts  @elastic/ml-ui
-# ML team owns the transform plugin, ES team added here for visibility
-# because the plugin lives in Kibana's Elasticsearch management section.
-/x-pack/plugins/transform/  @elastic/ml-ui @elastic/es-ui
```

Comment:
Hey CJ, sounds good to remove `es-ui`. However, this line will also remove `ml-ui`, please adapt to keep it. Could you also adapt the comment to something like `# ML team owns and maintains the transform plugin`?

Reply:
Whoops! Sorry, will fix this. 😊